Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2053,12 +2053,7 @@ extension BrowserViewController {
     }
     
     private func showProperIntroVC() {
-        // The onboarding screen type should always exist after
-        // the screen is presented for the 1st time
-        guard let onboardingScreenType = self.onboardingUserResearch?.onboardingScreenType else {
-            return
-        }
-        let introViewController = IntroViewControllerV2(onboardingType: onboardingScreenType)
+        let introViewController = IntroViewControllerV2()
```

Comment:
The viewmodel for introview controller should handle nil case and we no longer need to guard.